backend/src/Response/Admin/StoreOwnerDuesFromCashOrders/StoreOwnerDuesFromCashOrdersResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Response\Admin\StoreOwnerDuesFromCashOrders;
+
+use OpenApi\Annotations as OA;
+
+class StoreOwnerDuesFromCashOrdersResponse
+{
+   public int $id;
+
+   public string $storeOwnerName;
+
+   public int $orderId;
+
+   public float $amount;
+
+   public int $flag;
+     /**
+     * @OA\Property(type="object", property="createdAt")
+     */
+   public object $createdAt;
```

Comment:
here we can use DateTime as a var type instead of object and avoid using openAPI annotation.